index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap" rel="stylesheet">
+    <title>Samuel Felice | Full Stack Developer</title>
+</head>
+
+<body>
+    <header>
+        <a href="#" class="logo">My Logo</a>
+        <nav class="nav-icon">
+            <div class="icon"></div>
+            <div class="icon"></div>
+            <div class="icon"></div>
+        </nav>
+    </header>
+
+    <main>
+        <h1 class="primary-heading">I'm Samuel Glad to see you!</h1>
+
+        <p class="introductory-text">I’m a software developer! I can help you build a product , feature or website Look
+            through some of my work and experience! If you like what you see and have a project you need coded, don’t
+            hestiate to contact me.</p>
+
+        <h2 class="lets-connect">LET’S CONNECT</h2>
+
+        <ul class="social-media-icons">
+            <li><a href="#" class="icon-link"><img src="./images/icons/twitter.svg" alt="Twitter Logo"></a></li>
+            <li><a href="#" class="icon-link"><img src="./images/icons/linkedin.svg" alt="Linkedin Logo"></a></li>
+            <li><a href="#" class="icon-link"><img src="./images/icons/medium.svg" alt="Medium Logo"></a></li>
+            <li><a href="#" class="icon-link"><img src="./images/icons/github.svg" alt="Github Logo"></a></li>
+            <li><a href="#" class="icon-link"><img src="./images/icons/finger.svg" alt="Two Fingers"></a></li>
+        </ul>

```

Comment:
- [ ] **OPTIONAL** 
Great job on implementing the changes required by all the reviewers. though they missed out pointing this consider it as **[MANDATORY REQUIREMENT]** for this project and try implementing them.
- You should wrap all this in a container with a `section tag`  after your main tag, then you can control spacing around the section with margins (top, bottom, left, right) and padding(top, right, left, bottom) to control the contents inside it.